Rewrite (#179)
SW709E19/Report: report/PartDesign/ChapterDataStructure/DataStructure.tex

\subsection{Encoding}\label{sec:encoding}
The data structure should be encoded as samples.
A sample represents a collection of measurements associated with the same sensor, device, configuration, and gateway.
Samples should be stored as a JSON object, which is a common format supported by many programming languages, thus making it trivial to work with.
An example document can be seen in~\autoref{lst:datastructure}.

A number of measurements can be merged into a single JSON object instead of multiple objects where properties $time$ and $acceleration_x$ would be duplicated many times in the sample.
Merging measurements like this reduces the size of samples significantly.
The absolute size reduction corresponds approximately to $(n - 1) \times l$ where n is the number of measurements and l is the combined length of property names for a measurement.

> By using JSON as an encoding the amount of data transferred is reduces significantly and should thus be used within the implementation of the final system.

Reviewer: ```suggestion
By using this encoding the amount of data transferred is reduced significantly and should thus be used within the implementation of the final system.
```
